eclass/cmake.eclass

```diff
@@ -243,14 +243,44 @@ _cmake_modify-cmakelists() {
 	# Only edit the files once
 	grep -qs "<<< Gentoo configuration >>>" "${CMAKE_USE_DIR}"/CMakeLists.txt && return 0
 
+	local x re="VERSION( .*\.\.\.| )(([[:digit:]]+)\.([[:digit:]]+))"
```

Comment:
Can we do what we discussed and add the magic compat arg accordingly when we hit this case? i.e. QA notice + make it work?